app/com/lynxanalytics/biggraph/frontend_operations/ExportOperations.scala

```diff
@@ -153,4 +154,96 @@ class ExportOperations(env: SparkFreeEnvironment) extends OperationRegistry {
+    val nodesOrRelationships: String
+    override def enabled = FEStatus.enabled
+    override def trigger(wc: WorkspaceController, gdc: GraphDrawingController) = {
+      gdc.getComputeBoxResult(List(exportResult.gUID))
+    }
+    override def getOutputs(): Map[BoxOutput, BoxOutputState] = {
+      params.validate()
+      Map(context.box.output(
+        context.meta.outputs(0)) -> BoxOutputState.from(exportResult, params.toMap - "password"))
+    }
+    def exportResult() = {
+      val keys = splitParam("keys")
+      val attrs = (splitParam("to_export") ++ keys).toSet.toList
+      val t = graph_operations.AttributesToTable.run(
+        attrs.map(a => a -> project.vertexAttributes(a)))
+      assert(keys.nonEmpty, "You have to choose one or more attributes to use as the keys for identifying the nodes in Neo4j.")
+      val op = graph_operations.ExportAttributesToNeo4j(
+        params("url"), params("username"), params("password"), params("labels"), keys,
+        params("version").toInt, nodesOrRelationships)
+      op(op.t, t).result.exportResult
+    }
+  }
+
+  registerOp(
+    "Export vertex attributes to Neo4j", defaultIcon, ExportOperations,
+    List("graph"), List("exported"), new Neo4jAttributeExport(_) {
+      import Operation.Implicits._
+      params ++= List(
+        Param("labels", "Node labels", defaultValue = ""),
+        Choice("keys", "Attribute to use as key",
+          // Cannot join on internal ID ("<id>") and stuff like that.
+          options = project.vertexAttrList.filter(!_.id.startsWith("<")), multipleChoice = true),
+        Choice("to_export", "Exported attributes", options = project.vertexAttrList, multipleChoice = true))
+      val nodesOrRelationships = "nodes"
+    })
+  registerOp(
+    "Export edge attributes to Neo4j", defaultIcon, ExportOperations,
+    List("graph"), List("exported"), new Neo4jAttributeExport(_) {
+      import Operation.Implicits._
+      params ++= List(
+        Param("labels", "Relationship labels", defaultValue = ""),
+        Choice("keys", "Attribute to use as key",
+          // Cannot join on internal ID ("<id>") and stuff like that.
+          options = project.edgeAttrList.filter(!_.id.startsWith("<")), multipleChoice = true),
+        Choice("to_export", "Exported attributes", options = project.edgeAttrList, multipleChoice = true))
+      val nodesOrRelationships = "edges"
+    })
+
+  registerOp(
+    "Export graph to Neo4j", defaultIcon, ExportOperations,
+    List("graph"), List("exported"), new TriggerableOperation(_) {
+      import Operation.Implicits._
+      params ++= List(
+        Param("url", "Neo4j connection", defaultValue = "bolt://localhost:7687"),
+        Param("username", "Neo4j username", defaultValue = "neo4j"),
+        Param("password", "Neo4j password", defaultValue = "neo4j"),
+        NonNegInt("version", "Snapshot revision ID", default = 1),
+        Choice("node_labels", "Attribute with node labels",
```

Comment:
Mention that the value of this attribute can be a comma separated list of label values if user wants multiple labels. Would be more elegant to allow vectors here, but probably now is not the time.

Reply:
> Mention that the value of this attribute can be a comma separated list of label values if user wants multiple labels. Would be more elegant to allow vectors here, but probably now is not the time.

I expect you usually just use one label. Described the commas in the help.